samples/cpp/benchmark_app/main.cpp

```diff
@@ -427,6 +458,16 @@ int main(int argc, char* argv[]) {
                             // Use API 2.0 key for streams
                             key = ov::num_streams.name();
                             device_config[key] = ov::streams::AUTO;
+                        } else if (device == "MULTI" || device == "AUTO") {
```

Comment:
I think `MULTI` is not needed here, because it can not be specified without devices. It should work on the same way how for `HETERO`

Reply:
MULTI does not work if not specified without devices, like -d MULTI. This kind of case will encounter an exception throwed by MULTI plugin. However, the condition of this line will be reached if specified -d MULTI:xxx since the function parse_devices() at line 274 will select MULTI as the target device. This line will set the default nstreams value for hw device appeared in -d MULTI:xxx,xxx,...